src/code/z_actor.c

```diff
@@ -97,29 +132,114 @@ GLOBAL_ASM("asm/non_matchings/z_actor//func_800B6608.asm")
 
 GLOBAL_ASM("asm/non_matchings/z_actor//func_800B6680.asm")
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_MarkForDeath.asm")
+void Actor_MarkForDeath(Actor* actor) {
+    actor->draw = NULL;
+    actor->main = NULL;
+    actor->flags &= -2;
```

Comment:
Since this is a bitfield, it makes more sense to be explicit about what bits are affected.
```suggestion
    actor->flags &= ~0x1;
```

Reply:
Fixed